.gitignore

```
@@ -1,2 +1,3 @@
 .DS_Store
 css/excercise_one/part-c/css/excercise_one.css.map
+css/excercise_one/part-d/css/excercise_one.css.map
```

You can add for example 
```suggestion
css/excercise_one/part-c/css/excercise_one.css.map
css/excercise_one/part-d/css/excercise_one.css.map
*.map
```
so it will always add the files with that extension

---

Also we have a typo on the names of folders/files, it is exercise